R/run_ebola_simulations.R

```diff
@@ -0,0 +1,27 @@
+# Function to run multiple instances of sim_linelist
+run_ebola_simulations <- function(sim_count) {
+  # Initialize empty list to store simulation results
+  results <- vector("list", length = sim_count)
+
+  # Loop through simulations
+  for (i in 1:sim_count) {
+    # Set seed for reproducibility
+    set.seed(i)  # Use loop counter for unique seeds
+
+    # Run the simulation code and store the result
+    results[[i]] <- sim_linelist(
+      contact_distribution = contact_distribution,
+      infectious_period = ip_ebola_epidist,
+      onset_to_hosp = NA,
+      hosp_risk = NA,
+      hosp_death_risk = NA,
```

Comment:
Since the newest version of {simulist} these need to be set to `NULL` instead of `NA`.

Reply:
now changed